packages/grid/x-data-grid-pro/src/components/headerFiltering/GridHeaderFilterAdornment.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import InputAdornment from '@mui/material/InputAdornment';
+import {
+  GridFilterItem,
+  GridFilterOperator,
+  useGridApiContext,
+  GridColDef,
+  useGridRootProps,
+} from '@mui/x-data-grid';
+import { unstable_useId as useId } from '@mui/utils';
+import { unstable_gridHeaderFilteringMenuSelector } from '@mui/x-data-grid/internals';
+import { DataGridProProcessedProps } from '../../models/dataGridProProps';
+import { OPERATOR_SYMBOL_MAPPING } from './constants';
+
+function GridHeaderFilterAdornment(props: {
+  operators: GridFilterOperator<any, any, any>[];
+  field: GridColDef['field'];
+  item: GridFilterItem;
+  applyFilterChanges: (item: GridFilterItem) => void;
+  headerFilterMenuRef: React.MutableRefObject<HTMLButtonElement | null>;
+  buttonRef: React.Ref<HTMLButtonElement>;
+}) {
+  const { operators, item, field, buttonRef, headerFilterMenuRef, ...others } = props;
+
+  const buttonId = useId();
+  const menuId = useId();
+
+  const rootProps = useGridRootProps() as DataGridProProcessedProps;
+  const apiRef = useGridApiContext();
+  const open = Boolean(
+    unstable_gridHeaderFilteringMenuSelector(apiRef) === field && headerFilterMenuRef.current,
+  );
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    headerFilterMenuRef.current = event.currentTarget;
+    apiRef.current.showHeaderFilterMenu(field);
+  };
+
+  return (
+    <React.Fragment>
+      <InputAdornment position="start">
+        <rootProps.slots.baseIconButton
```

Comment:
Maybe make it a proper circle? Currently, it has an elliptic shape:


https://github.com/mui/mui-x/assets/13808724/b7b37735-f477-4711-a698-fc4180df3187

Reply:
Looks better now!
Although some symbols are not centered: 

<img width="135" src="https://github.com/mui/mui-x/assets/13808724/cbee614b-2314-4b24-9535-a98d11ff0f3f">